src/app/modules/common/types/app-types.d.ts

```diff
@@ -0,0 +1,49 @@
+export interface IUser {
+  name: string;
+  surname: string;
+  username: string;
+}
+
+export interface ITrip {
+  id: string;
+  name: string;
+  userId: string;
+  startDate: Date;
+  endDate: Date;
+}
+
+export interface IItem {
+  id: string;
+  tripId: string;
+  name: string;
+  tags: string[];
+  startTime: Date;
+  endTime: Date;
+  costEstimate: number;
+  currency: string;
+  startLocation: { lat: number; lon: number };
+  endLocation: { lat: number; lon: number };
+  notes: string;
+}
+
+export interface IPreferences {
+  userId: string;
+  currency: string;
+  interests: string[];
+  theme: string;
+}
+
+export interface ProfileCollection {
```

Comment:
I think the Interface naming should be kept consistent.

Reply:
The reason for the perceived inconsistency is because I'm trying to separate the responses from Firestore and what I will actually use within my state. So the types are consistent within their respective domains. Hope I am making sense